wallet/test/widget_testing/pages/purchase_item/widgets/pay_now_dialog_test.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/material.dart';
+import 'package:pylons_wallet/pages/purchase_item/widgets/pay_now_dialog.dart';
+import 'package:pylons_wallet/utils/image_util.dart';
+import '../../../../mocks/mock_constants.dart';
+import '../../../helpers/size_extensions.dart';
+
+void main() {
+  testWidgets('should find secondary pylon currency Icon', (tester) async {
```

Comment:
Kindly also add test when the swipe needs to be shown or when the add points need to be shown.